File: sumWithoutHighestAndLowestNumber.js
function sumArray(array) {
    if (array === null || array === []) {
        return 0
    } else {
        var array = array.sort(function(a, b) {return a-b})
        array.shift()
        array.pop()
        return array.reduce((accumulator, currentValue) => accumulator + currentValue, 0);
    }
}

console.log(sumArray([2]))

File: sumWithoutHighestAndLowestNumber.py
def sum_array(arr):
    if (arr == None or len(arr) == 0 or len(arr) == 1):
        return 0
    else:
        newArr = sorted(arr)
        newArr.pop(0)
        newArr.pop()
        return sum(newArr)

print(sum_array([3]))